semgrep-core/src/runner/Runner_config.ml

```diff
@@ -24,7 +34,8 @@ type t = {
   (* Main flags *)
   pattern_string : string;
   pattern_file : filename;
-  rules_file : filename;
+  rule_source : rule_source option;
+  lang_job : lang_job option;
```

Comment:
not a big fan of the _job suffix.

Reply:
It's a unit of work tied to a specific language. I was hoping it would be temporary until we no longer need to split the input by language, but I don't know.

Regarding the name: I don't know what else to call it. `job` is short and not too far off.